contracts/Safe.sol

```diff
@@ -300,8 +300,8 @@ contract Safe is
                 // When handling contract signatures the address of the contract is encoded into r
                 currentOwner = address(uint160(uint256(r)));
 
-                // Check that signature data pointer (s) is not pointing inside the static part of the signatures bytes
-                // This check is not completely accurate, since it is possible that more signatures than the threshold are send.
+                // Check that signature data pointer (s) is not pointing inside the static part of the signature bytes
```

Comment:
signatures referred to a variable here therefore it should have the s iirc

Reply:
Updated in #857